README.md

```diff
@@ -1 +1,6 @@
 README
+In this project, I have created a virtual pet, unit tested with JEST suit.
+
+Interact with this project by:
+1. git clone git@github.com:56aint/virtual-pet.git
```

Comment:
You could have a bit more of a fools guide here. I know when I read readmes, I like to have everything spelled out to me so it's really easy to pick up and get working.